### PR TITLE
feat: Backport interop test to rust-libp2p v049

### PR DESCRIPTION
**Pull request by thomaseizinger**
## Description

Version `0.49.0` does not support QUIC, WebRTC and TLS so I have removed those from the JSON file. What this effectively does is that we "pretend" as if the ping test is now "rust-libp2p-head". Meaning if the interop tests pass on this PR, we should be good to go with releasing this as a docker container and supplying the JSON file we see here to the test-plans repository.

## Change checklist

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] A changelog entry has been made in the appropriate crates

### CODE REVIEW

**Review by jxs — APPROVED** [review]
LGTM :)

File: Cargo.toml~ [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+    "transports/plaintext",
+    "transports/pnet",
+    "transports/tcp",
+    "transports/uds",
+    "transports/websocket",
+    "transports/wasm-ext",
+<<<<<<< HEAD
+=======
+    "transports/webrtc",
+>>>>>>> eeca244c (feat(interop-tests): build test binary inside container (#3441))
```
Comment (mxinden) [inline Cargo.toml~]: Merge conflict? Also is this file (`Cargo.toml~`) supposed to be checked in?

Comment (thomaseizinger) [reply Cargo.toml~]: It is not, not sure what went wrong there :thinking:

**Comment by mxinden** [comment]
Great. Thank you!

**Pull request merged**